### PR TITLE
fix(curriculum): Made output for sample code in snippet clearer - Multidimensional Array Access

### LINKED ISSUE #47220: Example Snippet Formatting
### Describe the Issue

The current example snippet for `JavaScript Algorithms and Data Structures` -> `Basic JavaScript` -> `Access Multi-Dimensional Arrays With Indexes` has the expected output underneath the sample code. It would be clearer for campers if the expected output is moved into the example snippet.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes

### Your code

Current Sample Snippet
```javascript
const arr = [
  [1, 2, 3],
  [4, 5, 6],
  [7, 8, 9],
  [[10, 11, 12], 13, 14]
];

arr[3];
arr[3][0];
arr[3][0][1];
```
`arr[3]` is `[[10, 11, 12], 13, 14]`, `arr[3][0]` is `[10, 11, 12]`, and `arr[3][0][1]` is `11`.

### Expected behavior

It would be more concise to give the output in the Snippet. 

### Screenshots

_No response_

### System

- Device: [Laptop]
- OS: [Windows 10]
- Browser: [Chrome]
- Version: []

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by mjh316**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #47220

### CODE REVIEW

**Review by hanswang123456 — COMMENTED** [review]
That looks better. It might be more consistent to assign them to a variable and put the output underneath the snippet.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.md [hunk 1]
```diff
@@ -21,13 +21,11 @@ const arr = [
   [[10, 11, 12], 13, 14]
 ];
 
-arr[3];
-arr[3][0];
-arr[3][0][1];
+arr[3]; // [[10, 11, 12], 13, 14]
+arr[3][0]; // [10, 11, 12]
+arr[3][0][1]; // 11
```
Comment (jeremylt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.md:26]: I really hate these dummy examples that do nothing when run. Perhaps something like this?

```suggestion
const subarray_3   = arr[3]; // [[10, 11, 12], 13, 14]
const subarray_3_0 = arr[3][0]; // [10, 11, 12]
const entry_3_0_1  = arr[3][0][1]; // 11
```

**Comment by hanswang123456** [comment]
Does this look better? @jeremylt This would make it consistent with (#46813).

**Review by hanswang123456 — COMMENTED** [review]
This does make it more consistent with the other challenges, I think it looks good.

**Review by jeremylt — DISMISSED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.md [hunk 2]
```diff
@@ -21,12 +21,13 @@ const arr = [
   [[10, 11, 12], 13, 14]
 ];
 
-arr[3];
-arr[3][0];
-arr[3][0][1];
+const subarray_3 = arr[3];
+const subarray_3_0 = arr[3][0];
+const entry_3_0_1 = arr[3][0][1];
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.md:26]: I think we should follow standard naming conventions here.

These should either be `subarray3`, `subarray30`, `entry301`, or `subarrayThree`, `subarrayThreeZero`, `entryThreeZeroOne`. I think I prefer the second.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.md (hunk 2 above)
Comment (jeremylt) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.md:26]: I think the first is unclear as to the indexing, which is why I added underscores.

I thought JS had the convention "don't use underscores unless it clearly improves readability".

I think using words for the numbers makes this harder to read.

maybe
```javascript
const subarray = arr[3];
const nestedSubarray = arr[3][0];
const element = arr[3][0][1];
```

Comment (naomi-lgbt) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.md:26]: That works even better, I think.

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me!

**Review by jeremylt — APPROVED** [review]

**Pull request merged**